arduino/include/arduino.h

```diff
@@ -0,0 +1,76 @@
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include <string>
+#include "serialib.h"
+#include <ios>
+#include <thread>
+#include <chrono>
+#include <mutex>
+#include <ctime>
+#include <cstdio>
+#include <stdexcept>
+#include <memory>
+class Arduino
+{
+public:
+    enum HEADERS
+    {
+        GET_FLOAT = 700,
+        GET_BOOLEAN = 701,
+        ASK_FOR_STATUS_PER_SENSOR = 30 /* the sensor*/,
```

Comment:
what this comment means?

Reply:
its the id of the sensor for example 301 for sensor 1